src/pages/patientView/pathwayMapper/PatientViewPathwayMapper.tsx

```diff
@@ -0,0 +1,291 @@
+import * as React from 'react';
+import _ from 'lodash';
+import { PatientViewPageStore } from '../clinicalInformation/PatientViewPageStore';
+import 'pathway-mapper/dist/base.css';
+import PathwayMapperTable, {
+    IPathwayMapperTable,
+    IPathwayMapperTableColumnType,
+} from '../../../shared/lib/pathwayMapper/PathwayMapperTable';
+import { observer } from 'mobx-react';
+import autobind from 'autobind-decorator';
+import { observable, computed, action } from 'mobx';
+import { Row } from 'react-bootstrap';
+
+import { AppStore } from 'AppStore';
+import { remoteData } from 'cbioportal-frontend-commons';
+import { fetchGenes } from 'shared/lib/StoreUtils';
+import 'react-toastify/dist/ReactToastify.css';
+import { ToastContainer } from 'react-toastify';
+import PatientViewUrlWrapper from '../PatientViewUrlWrapper';
+import { getGeneticTrackRuleSetParams } from 'shared/components/oncoprint/OncoprintUtils';
+import PathwayMapper, { ICBioData } from 'pathway-mapper';
+
+import 'cytoscape-panzoom/cytoscape.js-panzoom.css';
+import 'cytoscape-navigator/cytoscape.js-navigator.css';
+import 'react-toastify/dist/ReactToastify.css';
+
+const alterationFrequencyData: ICBioData[] = [];
+
+interface IPatientViewPathwayMapperProps {
+    store: PatientViewPageStore;
+    appStore: AppStore;
+    urlWrapper: PatientViewUrlWrapper;
+}
+
+const DEFAULT_RULESET_PARAMS = getGeneticTrackRuleSetParams(true, true, true);
+
+@observer
+export default class PatientViewPathwayMapper extends React.Component<
```

Comment:
This component still contains duplicate code from https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx. We should try to identify and eliminate duplicates. Ideally we should have a single customizable PathwayMapper component under the `shared` directory. That component then can be used in multiple views. Since we will also have another PathwayMapper for study view soon, it is not ideal to add yet another new component.

If it is too much work to refactor this component at this point, we may consider postponing it until we start working on the study view page.